src/con_duct/__main__.py

```diff
@@ -345,46 +345,46 @@ def execution_summary(self) -> dict[str, Any]:
             "exit_code": self.process.returncode,
             "command": self.command,
             "logs_prefix": self.log_paths.prefix,
-            "wall_clock_time": f"{self.elapsed_time:.3f} sec",
+            "wall_clock_time": f"{self.elapsed_time:.3f}",
             "peak_rss": (
-                f"{self.max_values.total_rss} KiB"
+                f"{self.max_values.total_rss}"
                 if self.max_values.stats
                 else "unknown"
             ),
             "average_rss": (
-                f"{self.averages.rss:.3f} KiB"
+                f"{int(self.averages.rss)}"
```

Comment:
if we are reporting bytes, i think `int` is preferable to `float`

Reply:
Yes, but might be worth checking if a number at all first